en/docs/install-and-setup/install/installation-prerequisites.md

```diff
@@ -87,9 +87,13 @@ The Micro Integrator runtime is tested with the following databases:
 
 | DBMS                 | Versions           |
 |----------------------|--------------------|
-| MySQL                | 5.7, 8             |
+| MySQL                | 9             |
 | Oracle               | 12c release 2, 19c |
 | Microsoft SQL Server | 2019               |
-| PostgreSQL           | 12.2, 13.2         |
+| PostgreSQL           | 17.2         |
 | MariaDB              | 10.5               |
 | DB2                  | 11.5               |
+
+### ARM Compatibility
+
+WSO2 Micro Integrator is compatible with ARM processors. It can run on ARM-based systems such as those with Apple Silicon or ARM-based Linux distributions.
```

Comment:
```suggestion
WSO2 Micro Integrator is compatible with ARM processors. It can run on ARM-based systems, such as those with Apple Silicon or ARM-based Linux distributions.
```